docker-alfresco/pom.xml

```diff
@@ -216,12 +216,12 @@
 
             <plugin>
                 <groupId>io.fabric8</groupId>
-                <artifactId>fabric8-maven-plugin</artifactId>
-                <version>${dependency.fabric8.version}</version>
+                <artifactId>docker-maven-plugin</artifactId>
```

Comment:
Do we know why the fabric8-maven-plugin was used in the first place? Changing to docker-maven-plugin doesn't buy us anything other than few less warning in the build logs, but maybe there was a reason for it in the first place?